src/baldr/edgeinfo.cc

```diff
@@ -9,10 +8,7 @@ namespace {
 
 // should return true if not equal to TaggedValue::kLinguistic
 bool IsNonLiguisticTagValue(char ch) {
-  static const std::unordered_set<TaggedValue> kNameTags =
-      {TaggedValue::kBridge, TaggedValue::kTunnel,   TaggedValue::kBssInfo, TaggedValue::kLayer,
-       TaggedValue::kLevel,  TaggedValue::kLevelRef, TaggedValue::kLandmark};
-  return kNameTags.count(static_cast<TaggedValue>(static_cast<uint8_t>(ch))) > 0;
+  return static_cast<TaggedValue>(ch) != TaggedValue::kLinguistic;
 }
```

Comment:
@gknisely Maybe I'm messing up with some lost sacred knowledge here, so please correct me if I'm doing a mistake.

Reply:
All tests are green. I just got confused by the inverse logic